File: starter_kit/bots/java/example_project/bots/MyBotUtils.java

package bots;
import pirates.game.PirateGame;

// A utility class to demonstrate the usage of another file in a project
public class MyBotUtils {

    // Returns true if the turn number divides by 100 with no remainder
    public static boolean isTurnMultipleOf100(PirateGame game) {
        return game.getTurn() % 100 == 0;
    }

}

File: starter_kit/bots/csharp/example_project/MyBotUtils.cs
using System;
using System.Collections.Generic;
using Pirates;


namespace Bots
{
    public class MyBotUtils
    {
        public static bool IsTurnMultipleOf100(IPirateGame game)
        {
            return game.GetTurn() % 100 == 0;
        }

    }
}
